content/about/_index.md

```diff
@@ -3,26 +3,30 @@ title: About
 layout: about
 ---
 
-### About Delphi
+### Who are we?
 
-Since 2012, we've been developing a [digital ecosystem to support](https://docs.google.com/presentation/d/13xwrzW17i1Hn_OvEe-8Ha8XZmexVkkIY22h1MO9QmcY/edit?usp=sharing) epidemic tracking and forecasting.  Pre-pandemic our focus was influenza; now it's COVID. We procure unique data streams that reflect COVID activity (from healthcare partners, tech companies, and massive national surveys), extract COVID-related indicators, and make these publicly and continuously available. These indicators are then used for nowcasting (situational awareness) and short-term forecasting.
+We're a research group based out of Carnegie Mellon University dedicated to developing the theory and practice of epidemic tracking and forecasting. Pre-pandemic our focus was influenza; now it's COVID. We procure unique data streams that reflect epidemic (or pandemic) activity, extract relevant indicators, and make these publicly and continuously available. These indicators are then used for nowcasting (situational awareness) and short-term forecasting.
 
 ### Who is our audience?
 
-Public health authorities (federal, state, local), fellow researchers (working on nowcasting and forecasting), fellow COVID response groups (e.g., COVID Exit Strategy), public and private sectors, data journalists, general public.
+Public health authorities (federal, state, local), fellow researchers (working on epidemic tracking and forecasting), public and private sectors, data journalists, and the general public.
 
-### What has our impact been?
+### Delphi's milestones
 
-Since 2013, we've supported CDC's Influenza Division in advancing and growing a scientific community around flu forecasting.  We've been perennial leaders in forecasting accuracy, and started providing flu nowcasts to state departments of health in 2016.
+- Since 2013, we've supported CDC's Influenza Division in advancing and growing a [scientific community around flu forecasting](https://www.cdc.gov/flu/weekly/flusight/index.html). We've been perennial leaders in forecasting accuracy.
 
-Since 2019, we've been working directly with CDC as a National Center of Excellence for Influenza Forecasting (a 5-year designation).
+- Since 2016, we've been TODO RONI please describe the Epidata API and you can [use this](https://cmu-delphi.github.io/delphi-epidata/) to link to it. 
```

Comment:
`TODO RONI`, @RoniRos FYI - we should get this content before this PR is merged.

Reply:
At 4:00 today I'm going to change this to:

> Since 2016, we've developed the [Epidata API](https://cmu-delphi.github.io/delphi-epidata/), which provides real-time access to epidemiological surveillance data.

At that point I think we can merge this in.